File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.py
from collections import Counter

# 두 문자열 입력
a = input()
b = input()

# 각 문자열의 문자 빈도수 계산
count_a = Counter(a)
count_b = Counter(b)

# 문자 차이 계산
diff = 0
for char in set(a + b):  # 두 문자열에 등장한 모든 문자
    diff += abs(count_a[char] - count_b[char])

# 결과 출력
print(diff)

File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.cc
#include <iostream>
#include <string>
#include <cmath> // abs() 사용
#include <map>
using namespace std;

int main(){
    string a,b;
    cin >> a >> b;
    
    // 각 문자열의 문자 빈도수 계산
    map<char,int> count_a, count_b;
    
    for (char c : a){
        count_a[c]++;
    }
    
    for (char c : b){
        count_b[c]++;
    }
    
    // 문자 차이 계산
    int diff = 0;
    for (char c = 'a'; c <= 'z'; c++){ //모든 소문자 알파벳에 대해
        diff += abs(count_a[c] - count_b[c]);
    }
    
    cout << diff << endl;
    return 0;
}